libs/lua-math-polygon/Makefile

```diff
@@ -0,0 +1,22 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lua-math-polygon
+PKG_VERSION:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/lua-math-polygon
+	SECTION:=libs
+	CATEGORY:=Libraries
+	TITLE:=Polygon math library can also convert 2 point rectangles into polygons
+endef
+
+define Build/Compile
+endef
+
+define Package/lua-math-polygon/install
+	$(INSTALL_DIR) $(1)/usr/lib/lua
+	$(INSTALL_DATA) src/math-polygon.lua $(1)/usr/lib/lua/
```

Comment:
The directory should not be called "src", as that has a special meaning (it will be copied to PKG_BUILD_DIR automatically).

Reply:
src is similar to the other lua libs in this repo. I was assuming It should be luasrc and it will be also nice to have the luasrcdiet feature here.